src/lib/selections.ts

```diff
@@ -141,6 +142,20 @@ export async function getEventForSelectWithPoint({
       },
     }
   }
+  if (_artifact.type === 'extrudeEdge') {
+    const codeRef = getExtrudeEdgeCodeRef(
+      _artifact,
+      engineCommandManager.artifactGraph
+    )
+    if (err(codeRef)) return null
```

Comment:
You don't want to propagate the error?

Reply:
I don't think so, no in this case.

Reply:
How will we ever know this was the error that occurred? (@jtran wants to eventually remove `console.error` from `err()`)

Reply:
Well here it's being used as more of a type guard/type narrowing.
But we could set up tracing?

Reply:
All good if you really think it shouldn't be passed on. After seeing you've made a couple passes I'd rather see this PR merged now to prevent rebase churn.